### DISCUSSION TITLE
verkefni/danski-johann-einelti

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#38] [vefsida-ummæli]
# verkefni/danski-johann-einelti

Danski Trausti leggur danska Jóhann í einelti

https://www.biskupsreglan.com/verkefni/danski-johann-einelti

**Comment by Ronaldo-is-better** [comment]
Drebe dig selv

**Reply by TJ478** [reply]
Ertu að tala illa við danska Trausta?
